bakery/src/scripts/fetch_update_metadata.py

```diff
@@ -27,6 +44,20 @@ def main():
     git_repo = Path(sys.argv[1]).resolve(strict=True)
     modules_dir = Path(sys.argv[2]).resolve(strict=True)
     repo = Repository(git_repo)
+    canonical_file = Path(sys.argv[3]).resolve(strict=True)
+    collection_dir = Path(sys.argv[4]).resolve(strict=True)
```

Comment:
The base commit for this branch is pretty old (probably why there are merge conflicts), but I think this parameter is already being passed in `master`.